qiskit_experiments/calibration_management/base_calibration_experiment.py

```diff
@@ -217,6 +218,24 @@ def _add_cal_metadata(self, experiment_data: ExperimentData):
         """
         pass
 
+    def _transpiled_circuits(self) -> List[QuantumCircuit]:
+        """Override the transpiled circuits method to bring in the inst_map.
+
+        The calibrated schedules are transpiled into the circuits using the instruction
+        schedule map which can be provided through the options. However, instances of
+        :class:`InstructionScheduleMap` are not serializable. This is fine since they
+        do not need to be serialized. Only instances of :class:`Calibrations` need to
+        be serialized. Here, we add the instruction schedule map to the transpile options
+        for transpilation and then remove it to avoid serialization issues.
+
+        Returns:
+            A list of transpiled circuits.
+        """
+        self.set_transpile_options(inst_map=self.calibrations.default_inst_map)
+        circs = super()._transpiled_circuits()
+        self.set_transpile_options(inst_map=None)
```

Comment:
You also need to manage this object too
https://github.com/Qiskit/qiskit-experiments/blob/fe8a0c49e21107c8d558d7c11cbdc7c4e63c1509/qiskit_experiments/framework/base_experiment.py#L72

Reply:
For this method it would probably be safer to not not set instance options and then reset them, and instead concretely list how you want to transpile for these experiments like:
```python
    def _transpiled_circuits(self) -> List[QuantumCircuit]:
        transpile_opts = copy.copy(self.transpile_options.__dict__)
        if "inst_map" in transpile_opts:
              LOG.warning("some warning here about overridding the set inst map option")
        transpile_opts["inst_map"] = self.calibrations.default_inst_map
        transpile_opts["initial_layout"] = list(self.physical_qubits)
        return transpile(self.circuits(), self.backend, **transpile_opts)
```

Reply:
Suggestion looks good. See here: [21ff9d8](https://github.com/Qiskit/qiskit-experiments/pull/684/commits/21ff9d8359ed88215f7f35bcd4ceba4152129cd5) and here [cde7d9a](https://github.com/Qiskit/qiskit-experiments/pull/684/commits/cde7d9ab1b56d81afb314636fce1d9e863cc152a)